mne/io/tag.py

```diff
@@ -341,9 +341,9 @@ def _read_simple(fid, tag, shape, rlims, dtype):
 
 def _read_string(fid, tag, shape, rlims):
     """Read a string tag."""
-    # Always decode to unicode.
+    # Always decode to ISO 8859-1 / latin1 (FIFF standard).
     d = _frombuffer_rows(fid, tag.size, dtype='>c', shape=shape, rlims=rlims)
-    return text_type(d.tostring().decode('utf-8', 'ignore'))
+    return text_type(d.tostring().decode('latin1', 'ignore'))
```

Comment:
FYI @agramfort I saw in the official docs that strings are written this way, not using UTF-8, so I have changed it :(

Reply:
[Yep](https://github.com/mne-tools/mne-python/blob/master/mne/io/fiff/tests/test_raw_fiff.py#L516-L525) though I just pushed a commit to more explicitly test it

Reply:
this will not break the IO of old files?

Reply:
It might make some of the characters look different. But either way (what we have in `master` vs what this PR does) some files are broken on read and write. The difference is that what this PR does follows the FIF spec, what we had before did not. So this should probably be in `whats_new.rst` under the `BUG` section.